torch_xla/csrc/helpers.cpp

```diff
@@ -619,4 +619,18 @@ xla::StatusOr<xla::XlaComputation> XlaHelpers::WrapXlaComputation(
   return builder.Build(orig_result);
 }
 
+torch::lazy::Shape XlaHelpers::ConvertXlaShapeToLazy(const xla::Shape& shape) {
+  at::ScalarType scalar_type = TensorTypeFromXlaType(shape.element_type());
```

Comment:
It looks like `TensorTypeFromXlaType` could fail. Should we have a test case for that?

Reply:
Good point, that was also on my mind but I thought that kind of testing should be unit-tested on that `TensorTypeFromXlaType` instead of our helper method. But looking back, it seems like we don't have any unit tests on that `TensorTypeFromXlaType` helper method. I'll add a test for the failure case.